### PR TITLE
stlsplit 1.1 (new formula)

### PR DESCRIPTION
**Pull request by hroncok**

### CODE REVIEW

File: Library/Formula/stlsplit.rb [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  depends_on "admesh"
+
+  def install
+    system "premake4", "gmake"
+    system "make"
+    # there is no make install
+    mkdir "#{bin}"
+    mkdir "#{lib}"
+    mkdir "#{include}"
+    cp "build/stlsplit", "#{bin}"
```
Comment (DomT4) [inline Library/Formula/stlsplit.rb]: You can replace all of the ` mkdir `s and ` cp `s with ` bin.install "build/stlsplit" ` and similar.

File: Library/Formula/stlsplit.rb [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+      facet normal -1 -0  0
+        outer loop
+          vertex  0  1  1
+          vertex  0  0  0
+          vertex  0  0  1
+        endloop
+      endfacet
+    endsolid  admesh
+    EOS
+    system "stlsplit", "test.stl"
```
Comment (DomT4) [inline Library/Formula/stlsplit.rb]: `system bin/"stlsplit", "test.stl"`

**Comment by hroncok** [comment]
Thanks @DomT4, updated, amended.

**Comment by mikemcquaid** [comment]
Failing on 10.8, 10.9: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/24546/version=mavericks/testReport/junit/brew-test-bot/mavericks/install_stlsplit/

**Comment by hroncok** [comment]
Oh, parallel build didn't work, I hope I've fixed this upstream. (Updated the formula to 1.1)

**Comment by mikemcquaid** [comment]
Oh, I didn't realise this was a self-submission. Clicking through to the project page this project looks a little too niche for us I'm afraid. I'm happy for you to keep getting feedback and CI here but it's probably best if you create your own tap.

**Comment by hroncok** [comment]
I'm sorry, I've never meant to use you as CI or anything similar. This is my first submission to homebrew and I just wanted this simple tool to be available from there. I've just forgot to test the parallel build.

Is there anything I should hove done differently? Is it undesirable to submit my own projects?

**Comment by DomT4** [comment]
The [Acceptable Formulae guide](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Acceptable-Formulae.md#niche-or-self-submitted-stuff) prohibits self-submissions unless the project has some significant wider popularity. Just ensures formulae are kept updated and maintained and such.

Best thing to do is create a Homebrew tap and provide it there. If it gains popularity over time, you're welcome to resubmit it, or someone else can submit it if they find it useful :).

**Comment by hroncok** [comment]
Thanks for explanation and sorry for not reading this part of documentation before.

**Comment by mikemcquaid** [comment]
No need to apologise @hroncok, sorry we can't accept it at this time.

**Pull request closed without merging**